modules/troubleshooting-common-problems-with-distributed-workloads-for-users.adoc

```diff
@@ -0,0 +1,234 @@
+:_module-type: REFERENCE
+
+[id="troubleshooting-common-problems-with-distributed-workloads-for-users_{context}"]
+= Troubleshooting common problems with distributed workloads for users
+
+[role='_abstract']
+If you are experiencing errors in {productname-long} relating to distributed workloads, read this section to understand what could be causing the problem, and how to resolve the problem.
+
+ifndef::upstream[]
+If you cannot see the problem here or in the release notes, contact {org-name} Support.
```

Comment:
```suggestion
If you do not see the problem here or in the release notes, contact {org-name} Support.
```
Perhaps? "Cannot" implies ability whereas "do not" implies it just not existing here

Reply:
I see your point, but the suggested rewrite is also ambiguous in that it could be misinterpreted as "If you do not experience the problem ...", so I rephrased as follows:
"If the problem is not documented ..."

I think the passive voice is acceptable in this case.

Reply:
I totally agree - love the new rewrite. Thanks!